changelog.d/README.rst

```diff
@@ -0,0 +1,94 @@
+.. _Adding change notes with your PRs:
+
+Adding change notes with your PRs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is very important to maintain a log for news of how
+updating to the new version of the software will affect
+end-users. This is why we enforce collection of the change
+fragment files in pull requests as per `Towncrier philosophy`_.
+
+The idea is that when somebody makes a change, they must record
+the bits that would affect end-users only including information
+that would be useful to them. Then, when the maintainers publish
+a new release, they'll automatically use these records to compose
+a change log for the respective version. It is important to
+understand that including unnecessary low-level implementation
+related details generates noise that is not particularly useful
+to the end-users most of the time. And so such details should be
+recorded in the Git history rather than a changelog.
+
+Alright! So how do I add a news fragment?
```

Comment:
Consider using third-person imperative voice (no first-person "I" or "we" or "you", but instead "do this" and "how to do that").

Reply:
Is it a style preference or am I doing English wrong? :)

Reply:
It's a style preference, but a recommended one for technical writing. I'm happy to do the edits if you prefer.

Reply:
I'll edit it, it's fine. I just wanted to clarify all the things before updating this PR altogether.

Reply:
I'm a bit confused. I've always thought that technical/academic writing encourages "we"-phrasing... Also, is this comment about the title specifically or do you also want some changes elsewhere?

Reply:
I know first-person-plural ("we") is not uncommon, especially in tech, but it's not crucial. I'm happy to accept with 3rd-person plural and I'll edit later. This comment was just a nitpick and can be ignored.